### ISSUE TITLE
Change base es target to es2020

### ISSUE THREAD

**Issue by pokey** [cursorless-dev/cursorless#2484]
in tsconfig.base.js. And then override as es6 or whatever where we actually need that

**Comment by pokey** [comment]
thought about this one a bit more, and I think the idea here was that the base config should be the min es target of all of our entrypoint packages, so that the libs can be imported anywhere. Does that make sense @auscompgeek ?

**Comment by auscompgeek** [comment]
Yep, I think that was the reason we didn't bump the base target in the first place.
